zerver/lib/webhooks/git.py

```diff
@@ -147,3 +152,14 @@ def get_commits_content(commits_data, is_truncated=False):
 def get_short_sha(sha):
     # type: (Text) -> Text
     return sha[:7]
+
+def get_all_committers(user_name, commits_data):
+    # type: (Text, List[Dict[str, Any]]) -> (Dict[str, Any], str)
+    committers = defaultdict(int)
+    others = ''
+    for commit in commits_data:
+        committers[commit.get('name')] += 1
+    for name, no_of_commits in committers.items():
+        if name != user_name:
+            others = ' '.join((others, OTHER_COMMITTERS.format(name=name, no_of_commits=no_of_commits)))
```

Comment:
I see use `join` where you can use just format function.
in such cases please use:
`"{} {} {}".format(others,....)`